pkg/kn/commands/eventtype/create.go

```diff
@@ -82,6 +104,14 @@ func NewEventtypeCreateCommand(p *commands.KnParams) *cobra.Command {
 	}
 	commands.AddNamespaceFlags(cmd.Flags(), false)
 
+	referenceFlag.AddWithFlagName(cmd, "reference", "r")
+	flag := "reference"
+	cmd.Flag(flag).Usage = "Addressable Reference producing events. " +
+		"You can specify a broker, channel, or fully qualified GVK resource. " +
+		"Examples: '" + flag + " broker:nest' for a broker 'nest', " +
+		"'" + flag + " channel:pipe' for a channel 'pipe'."
+	//TODO: craft fqdn GVK example.
```

Comment:
Important to not forget it here, since using a GVK is the most complex way to specify this reference so there should be an example for sure.

Reply:
From my e2e tests it seems there's an issue with GVK. At least it's not working for `EventTypes`. I'll be certainly taking a bit in-depth look.